File: Nodejs/backjoonNode/string/27866.js
/*
백준 27866: 문자와 문자열
https://www.acmicpc.net/problem/27866
*/

const fs = require('fs');
let [strInput, strIdx] = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

console.log(strInput[parseInt(strIdx)-1]);

File: C/백준/Step5/27866.c
// Sol 1
#include <stdio.h>

int main () {
    char strInput[1001];
    int strIdx;
    
    scanf("%s", strInput);
    scanf("%d", &strIdx);
    
    printf("%c", strInput[strIdx-1]);
    
    return 0;
}

// Sol 2